src/commands/new.ts

```diff
@@ -34,7 +34,7 @@ export default class New extends Command {
     const fileName = flags['file-name'] || DEFAULT_ASYNCAPI_FILE_NAME;
     const template = flags['example'] || DEFAULT_ASYNCAPI_TEMPLATE;
 
-    this.createAsyncapiFile(fileName, template);
+    await this.createAsyncapiFile(fileName, template);
```

Comment:
Actually this is what is fixing it

Reply:
Oh, I got it 🙂

Do I need to merge this PR with my current branch and then push or simply push my local changes to the current one?

Reply:
Yeah just push your local changes, just double check if everything is alright.

Reply:
Okay, thank you so much for helping me out 😊